apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/components/amount_selection/BaseAmountBox.java

```diff
@@ -155,23 +169,36 @@ private View(Model model, Controller controller) {
             baseAmountInfoIcon.setMinWidth(Label.USE_PREF_SIZE);
 
             HBox.setMargin(baseAmountInfoIcon, new Insets(0, 0, 0, 4));
-            root.getChildren().addAll(baseAmountLabel, codeLabel, baseAmountInfoIcon);
+            HBox.setMargin(bitcoinAmountDisplay, new Insets(0, 7, 0, 0));
+
+            root.getChildren().addAll(baseAmountLabel, codeLabel, bitcoinAmountDisplay, baseAmountInfoIcon);
             root.setAlignment(Pos.CENTER);
             root.getStyleClass().add("base-amount-box");
 
             amountListener = this::onAmountChanged;
+            isBtcListener = this::onIsBtcChanged;
+            codeListener = this::onCodeChanged;
         }
 
         @Override
         protected void onViewAttached() {
+            boolean isBtc = model.isBtc.get();
```

Comment:
isBtc is used only once in the recent version. No need to extract it as local var.